modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -40,324 +40,467 @@
-        "Focus Camera",
-        "Focus camera on parent."
+    constexpr openspace::properties::Property::PropertyInfo LineWidthInfo = {
+        "LineWidth",
+        "Line Width",
+        "This value specifies the line width of the field lines if the "
+        "selected rendering method includes lines."
     };
     constexpr openspace::properties::Property::PropertyInfo TimeJumpButtonInfo = {
-        "timeJumpToStart",
+        "TimeJumpToStart",
         "Jump to Start Of Sequence",
         "Performs a time jump to the start of the sequence."
     };
 
-    enum class SourceFileType : int {
-        Cdf = 0,
-        Json,
-        Osfls,
-        Invalid
-    };
+    struct [[codegen::Dictionary(RenderableFieldlinesSequence)]] Parameters {
+        enum class SourceFileType {
+            Cdf,
+            Json,
+            Osfls
+        };
+        // Input file type. Should be cdf, json or osfls
+        SourceFileType inputFileType;
 
-    float stringToFloat(const std::string& input, float backupValue = 0.f) {
-        float tmp;
-        try {
-            tmp = std::stof(input);
-        }
-        catch (const std::invalid_argument& ia) {
-            LWARNING(fmt::format(
-                "Invalid argument: {}. '{}' is NOT a valid number", ia.what(), input
-            ));
-            return backupValue;
-        }
-        return tmp;
-    }
+        // Should be path to folder containing the input files
+        std::filesystem::path sourceFolder [[codegen::directory()]];
+
+        // Path to a .txt file containing seed points. Mandatory if CDF as input.
+        // Files need time stamp in file name like so: yyyymmdd_hhmmss.txt
+        std::optional<std::filesystem::path> seedPointDirectory [[codegen::directory()]];
+
+        // Currently supports: batsrus, enlil & pfss
+        std::optional<std::string> simulationModel;
+
+        //sim mod
+        //std::optional<openspace::fls::Model> model;
+
+        // Extra variables such as rho, p or t
+        std::optional<std::vector<std::string>> extraVariables;
+        
+        // Which variable in CDF file to trace. b is default for fieldline
+        std::optional<std::string> tracingVariable;
+
+        // 1.f is default, assuming meters as input.
+        // Convert the models distance unit, ex. AU for Enlil, to meters.
+        // Can be used during runtime to scale domain limits.
+        std::optional<float> scaleToMeters;
+
+        // If False (default) => Load in initializing step and store in RAM
+        std::optional<bool> loadAtRuntime;
+
+        // Values should be paths to .txt files
+        std::optional<std::vector<std::string>> colorTablePaths;
+
+        // [[codegen::verbatim(ColorMethodInfo.description)]]
+        std::optional<std::string> colorMethod;
+
+        // [[codegen::verbatim(ColorQuantityInfo.description)]]
+        std::optional<int> colorQuantity;
+
+        // Values should be entered as {X, Y}, where X & Y are numbers
```

Comment:
Again, needs some more description about what it actually is. Maybe "Min and max ranges for each of the color tables, given in the same order as the paths to the color tables."

But yeah, gonna stop commenting on these descriptions now. Please give them a little more love  so the users understand what the variables actually are 😄 It is important to give an understanding of how the renderable works as well